doc/ipython-notebooks/clustering/GMM.ipynb

```diff
@@ -551,9 +527,7 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
```

Comment:
in order to avoid all these unrelated changes in the diff, could you first send a PR where do dont change anything, but rather just save the notebook, and then do a second Pr with your changes.
But only do it next time, this one is OK